### ISSUE TITLE
sometimes the module fails to get the download link

### ISSUE THREAD

**Issue by advith231** [mansuf/zippyshare-downloader#3]
![image](https://user-images.githubusercontent.com/81084208/112463483-51a88780-8d88-11eb-8268-d1dc3620d250.png)
If the module function fails the download size of the file will be less than 1MB but the problem is even if the module function fails to get the direct download link it will give file size as 500 (I will use recursive function to recheck since zippyshare website will change every time we visit)
Code used is shown below

```python
import  re
from zippyshare_downloader import Zippyshare




def size_check(txt):
 
 
  z = Zippyshare()
  info=z.extract_info(txt,0)
  print(info["size"])
  a3=0
  temp = re.compile("([0-9]+)")
  ras = temp.match(info["size"]).groups()
  a3=int(ras[0])
 
  if(a3>1):
     z.extract_info(txt)
  else :
    size_check(txt)


txt =input("test")

x = txt.split()

for i in range(len(x)):
  size_check(x[i])
```

**Comment by mansuf** [comment]
first of all, thank you for giving this important information !

> If the module function fails the download size of the file will be less than 1MB 

yes, i just found a bug in the module, that sometimes the module incorrectly process the random number in download link. So the download link will redirect you to Zippyshare website instead downloading the file

> but the problem is even if the module function fails to get the direct download link it will give file size as 500.

well, it doesn't matter. The **file size** is grabbed from zippyshare website itself, not from the downloaded zippyshare file in your Computer

Currently im working to fix this, the update will release soon (v0.0.15).

**Comment by mansuf** [comment]
The issue has been fixed and update has been released, please update your `zippyshare-downloader` to the latest version (v0.0.15)

by the way, if you like this module. Please give a star to this repository :D
